hello.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
- [ ] Excellent job creating a file to add styles for your project 💯, but if you check your website your styles are not having effect.
![image](https://user-images.githubusercontent.com/91301423/196821705-83c77ded-04a4-4344-bb23-bf928a39a1db.png)
You should create a reference to this file inside of the `<head>` tag, please add this line of code 
`<link rel="stylesheet" href="style.css">` and check if the changes are done 🤪 .